src/core/commands.ts

```diff
@@ -414,14 +408,6 @@ export const updateProjectProps = async (payload: {
   await triggerInternal('ProjectChanged', { project: response.project })
   return
 }
-/**
- * @deprecated Use updateProjectProps
- */
-export const updateProjectMeta = (payload: {
-  projectId: SDK.Project['id']
-  /** Arbitrary metadata to associate with this project */
-  meta?: Props
-}) => updateProjectProps({ projectId: payload.projectId, props: payload.meta })
```

Comment:
This method is referenced in  host.tsx in example project

Reply:
That's a good point - I'll fix that. moving forward, I don't think we should worry if the example project falls out of sync since it's not being used. we'll probably need to revisit it if we start marketing studio kit again